src/test/java/org/fundacionjala/pivotal/steps/api/APIProjectSteps.java

```diff
@@ -32,26 +44,17 @@ public void iPostANewProjectToEndpoint(final String endpoint, final Map<String,
      */
     @And("^save the response as \"([^\"]*)\"$")
     public void saveTheResponseAs(final String variableName) {
+       // date.setDate(date.generateDate());
+       // System.out.println(date.getDate());
         APICommons.saveResponse(variableName, APIRequestManager.getResponse());
-        APICommons.buildEndPoint(APIRequestManager.getResponse());
+        //APICommons.buildEndPoint(APIRequestManager.getResponse());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)